Makefile

```diff
@@ -161,17 +163,11 @@ manifests:
 
 images: openstack-cluster-api-controller clusterctl-image manifests
 
-openstack-cluster-api-controller: depend manager manifests
-ifeq ($(GOOS),linux)
-	cp bin/manager cmd/manager
-	docker build -t $(REGISTRY)/openstack-cluster-api-controller:$(VERSION) cmd/manager
-	rm cmd/manager/manager
-else
-	$(error Please set GOOS=linux for building the image)
-endif
+openstack-cluster-api-controller: generate fmt vet manifests
+	docker build . -f cmd/manager/Dockerfile --network=host -t "$(REGISTRY)/openstack-cluster-api-controller:$(VERSION)"
```

Comment:
Why ` --network=host`?

Reply:
the docker image required to install package for apt repo, and the docker image bydefault use the default network to build docker image which may not able to access the internet.

Reply:
If the container cannot access internet, then the host may not able to access internet either?

Reply:
there should be the DNS problem, the host can solve the deb.debian.org but the container can not.

here is the message from containers:
```
Err:1 http://deb.debian.org/debian stretch InRelease
  Temporary failure resolving 'deb.debian.org'
Err:2 http://security.debian.org/debian-security stretch/updates InRelease
  Temporary failure resolving 'security.debian.org'
```

Reply:
FYI https://stackoverflow.com/questions/27435479/pass-net-host-to-docker-build

/approve